cmd/plan/pl005/plan.go

```diff
@@ -7,36 +7,32 @@ import (
 )
 
 // Plan describes in which order and with which tolerance to execute actions of
-// this cluster scope.
+// this test plan.
 var Plan = []plan.Step{
 	{
-		Action:  "ac001",
+		Action:  "create/cluster/singlecontrolplane",
 		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
-		Comment: "create cluster CRs",
 	},
 	{
-		Action:  "ac002",
+		Action:  "verify/cluster/created",
 		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
-		Comment: "check cluster access",
 	},
 	{
```

Comment:
Do we want to add node pools to the upgrade? I remember that you mentioned there is something off if there is only one master and the update is super quick. I added the node pool creation and verification to the patch upgrade test.

Reply:
hmm yeah, I will add that

What do you think about:
- Create Control Plane
- Create NodePool
- Check Control Plane
- Check NodePool

This would save quite a bit of time as CF stacks are created in parallel compared to:
- Create Control Plane
- Check Control Plane
- Create NodePool
- Check NodePool

Reply:
Makes sense. We can then check that all plans follow that pattern where desired.

Reply:
yeah, will make sure about it in a different PR were I fix all plans